File: FangZhiHu/app/src/main/java/com/tongqinyuan/fangzhihu/HomeContact.java
package com.tongqinyuan.fangzhihu;


/**
 * This specifies the contract between the view and the presenter.
 * Created by liyanxi on 11/14/17.
 * Copyright (c) 2017 www.zhengshijr.com. All rights reserved.
 */

public interface HomeContact {

    interface View {
        void requestDataSuccess(Object homeDetailEntity);
    }

    interface Presenter  {
        void getHomeData();

    }
}

File: FangZhiHu/KotlinOKhttp/src/main/java/com/kotlinokhttp/HomeContact.kt
package com.kotlinokhttp

import com.ytq.main.common.IPresenter

/**
 * Created by tongqinyuan on 2018/3/9.
 */
interface HomeContact {

    interface View {
        fun requestDataSuccess(homeDetailEntity: Any)
    }

    interface Presenter :IPresenter {
        fun getHomeData()

    }
}